application/F3DOptionsParser.cxx

```diff
@@ -65,34 +65,14 @@ class ConfigurationOptions
 protected:
   bool GetOptionConfig(const std::string& option, std::string& configValue) const
   {
-    bool ret = false;
-    if (this->FilePathForConfigBlock.empty())
+    auto localIt = this->CurrentFileConfig.find(option);
+    if (localIt != this->CurrentFileConfig.end())
     {
-      auto localIt = this->GlobalConfigEntry.find(option);
-      if (localIt != this->GlobalConfigEntry.end())
-      {
-        configValue = localIt->second;
-        ret = true;
-      }
+      configValue = localIt->second;
+      return true;
     }
-    else
-    {
-      for (auto const& it : this->RegexConfigEntries)
-      {
-        std::regex re(it.first, std::regex_constants::icase);
-        std::smatch matches;
-        if (std::regex_match(this->FilePathForConfigBlock, matches, re))
-        {
-          auto localIt = it.second.find(option);
-          if (localIt != it.second.end())
-          {
-            configValue = localIt->second;
-            ret = true;
-          }
-        }
-      }
-    }
-    return ret;
+    // TODO ensure to return the actual default value if option key is not found
```

Comment:
this TODO doesnt seems necessary, since this call is made with the default value

Reply:
that's leftover from my attempt at handling `null` values, I forgot to remove it